internal/indexmanagement/scripts.go

```diff
@@ -131,6 +131,45 @@ def updateWriteIndex(currentIndex, nextIndex, alias):
     print(e)
     sys.stdout = original_stdout
     return False
+
+def getCurrentDiskPercent():
+  original_stdout = sys.stdout
+  try:
+    es_client = getEsClient()
+    response = es_client.cat.allocation(format="JSON", h="disk.percent,node", v=True)
+    diskPercent = []
+    for i in range(len(response)):
+      diskPercent.append(int(response[i]["disk.percent"]))
+    currentDiskPercent = max(diskPercent)
+    return currentDiskPercent
+  except Exception as e:
+    sys.stdout = open('/tmp/response.txt', 'w')
+    print(e)
+    sys.stdout = original_stdout
+    return False
+
+def deleteByPercentage(alias):
+  original_stdout = sys.stdout
+  try:
+    es_client = getEsClient()
+    indices = es_client.cat.indices(index=alias+"*",format="json",h="index,creation.date")
+
+    currentDiskPercent = getCurrentDiskPercent()
+    diskThreshold = os.environ['DISK_THRESHOLD']
```

Comment:
Lets pass `diskThreshold` as a variable to `deleteByPercentage()`
What I mean is `def deleteByPercentage(alias, diskThreshold):`